src/utils/cloneDeep.ts

```diff
@@ -0,0 +1,3 @@
+export function cloneDeep<T>(obj: T): T {
+  return JSON.parse(JSON.stringify(obj))
+}
```

Comment:
You can use [`structuredClone`](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone), which is already included in the polyfill.